package/cfg/rke2-cis-1.5/policies.yaml

```diff
@@ -0,0 +1,239 @@
+---
+controls:
+version: 1.5
+id: 5
+text: "Kubernetes Policies"
+type: "policies"
+groups:
+- id: 5.1
+  text: "RBAC and Service Accounts"
+  checks:
+    - id: 5.1.1
+      text: "Ensure that the cluster-admin role is only used where required (Not Scored)"
+      type: "manual"
```

Comment:
We have policies audit scripts implemented for rke, they are pretty generic kubectl commands, and I think rke2 can re-use those instead of calling these as  type: "manual"